src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -154,32 +131,41 @@ function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, u
                 />
             </OfflineWithFeedback>
         );
-    }
+    };
 
     /**
      * Add free policies (workspaces) to the list of menu items and returns the list of menu items
-     * @returns {Array} the menu item list
      */
     const workspaces = useMemo(() => {
-        const reimbursementAccountBrickRoadIndicator = !_.isEmpty(reimbursementAccount.errors) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : '';
+        const reimbursementAccountBrickRoadIndicator = !_.isEmpty(reimbursementAccount?.errors) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : '';
         return _.chain(policies)
```

Comment:
Let's re-write this without lodash (I think we may have to in order to get the typescript match correctly)